configure/vault-license.tf

```diff
@@ -0,0 +1,14 @@
+# Vault license
+resource "null_resource" "vault-license" {
+  provisioner "local-exec" {
+    command = <<EOF
+gcloud container clusters get-credentials "${google_container_cluster.vault.name}" --zone="${google_container_cluster.vault.zone}" --project="${google_container_cluster.vault.project}"
+
+# install Vault license (which is mounted into Vault container via k8s secrets)
+kubectl exec vault-cluster-0 -- \
+  curl -X PUT \
+    -H "X-Vault-Token: ${var.vault_token}" \
+    --cacert /etc/vault/tls/ca.pem \
+    --data @/etc/vault/data/vault.license \
+    https://localhost:8200/v1/sys/license
+}
```

Comment:
@andredumas Literally have no idea how I managed to truncate two lines from this file before committing. I tested it thoroughly!